pyproject.toml

```diff
@@ -0,0 +1,6 @@
+[tool.bandit]
+exclude_dirs = ["bin", "lib"]
+skips =["B105","B106","B603","B404"]
```

Comment:
Rather than blanket skips, consider using `# nosec` comments, per bandit docs: https://bandit.readthedocs.io/en/latest/config.html#suppressing-individual-lines